xml/manager_modules.xml

```diff
@@ -185,4 +185,68 @@
    </tip>
   </sect2>
  </sect1>
+ <sect1 xml:id="mgr-modules-telemetry">
+  <title>Telemetry Module</title>
+  <para>
+    The telemetry plugin sends anonymous data about the cluster, in which it
+    is running, back to the Ceph project.</para>
+  <para>
+    This (opt-in) component contains counters and statistics on how the cluster has been
+    deployed, the version of Ceph, the distribition of the hosts and other
+    parameters which help the project to gain a better understanding of the
+    way Ceph is used. It does not contain any sensitive data like pool names,
+    object names, object contents or hostnames.</para>
+  <para>The purpose of the telemetry module is to provide an automated feedback
+    loop for the developers to help quantify adoption rates, tracking, or
+    pointing out things that need to be better explained or validated during
+    configuration to prevent undesirable outcomes.</para>
+  <note>
+    <para>
+      The Telemetry Module requires the mgr nodes to have the ability to push
+      data over HTTPS to the upstream servers. Ensure your corporate firewalls
+      permit this action.
+    </para>
+  </note>
+  <para>If you would like to read more, or see some of the early findings
+    of using this tool, we recommend reading <link xlink:href="https://ceph.io/community/the-first-telemetry-results-are-in/"/>.
+  </para>
+  <procedure>
+    <step>
+      <para>
+        To enable the telemetry module:
+      </para>
+      <screen>ceph mgr module enable telemetry</screen>
+      <note>
+        <para>
+          This command only enables you to view your data locally. This
+          command does not share your data with the Ceph community.
+        </para>
+      </note>
+    </step>
+    <step>
+      <para>To allow the telemetry module to start sharing data:</para>
+      <screen>ceph telemetry on</screen>
+    </step>
+    <step>
+      <para>To disable telemetry data sharing:</para>
+      <screen>ceph telemetry off</screen>
+    </step>
+    <step>
+      <para>To generate a JSON report that can be printed:</para>
+      <screen>ceph telemetry show</screen>
+    </step>
+    <step>
+      <para>To add a contact and description to the report:</para>
+      <screen>ceph telemetry config-set contact ‘John Doe <literal>john.doe@example.com</literal>’
+        ceph telemetry config-set description ‘My first Ceph cluster’</screen>
```

Comment:
I think these two want to be in the `ceph config set mgr mgr/telemetry/contact` and `.../description` form, like interval is (but I can't test right this instant, because I've mangled my dev environment)

Reply:
This is direct from ceph.io. Let me know if the config is different with SES and I can change.

Reply:
I did some more digging, it was `config-set` in the mimic (13.x) release (https://docs.ceph.com/docs/mimic/mgr/telemetry/) that form was dropped in nautilus (14.x) (https://docs.ceph.com/docs/nautilus/mgr/telemetry/), which is what SES6 is based on. Also I tested it now that I've unmangled my dev environment ;-)